libflux/src/core/semantic/builtins.rs

```diff
@@ -872,6 +872,14 @@ pub fn builtins() -> Builtins<'static> {
                 "null" => r#"forall [t0] t0"#,
                 "none" => r#"forall [t0] t0"#,
             },
+            "contrib/jsternberg/rows" => semantic_map! {
+                "map" => r#"
+                    forall [t0, t1, t2, t3] where t0: Row, t1: Row, t2: Row, t3: Row (
+                        <-tables: [t0],
+                        fn: (r: t2) -> t3
+                    ) -> [t1]
+                "#,
```

Comment:
This seems like too many type variables. Shouldn't the type signature be the same as the regular map function?

    forall [t0, t1] (<-tables: [t0], fn: (r: t0) -> t1) -> t1

Reply:
I didn't think about that. Fixed.